tests/fixtures/generic_models.py

```diff
@@ -1,11 +1,11 @@
 from typing import Generic, TypeVar
 
-from pydantic.generics import GenericModel
+from pydantic import BaseModel
```

Comment:
Indeed, doesn't look like a lot of changes :)

But I do want to be able to target both **pydantic** `v1` and `v2` models when translating to **zod** declarations.

I'll tinker with this tomorrow.

Thanks 🙌